src/JsonViewer.ts

```diff
@@ -84,6 +100,32 @@ export class JsonViewer extends LitElement {
         this.setState(toggleNode(path));
     };
 
+    handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
because up and down normally scroll the page, this method should `event.preventDefault()` to not wander away from the JSON

Reply:
Good catch

Changes applied